neurons/miner.py

```diff
@@ -124,10 +129,50 @@ def check_version(self, query):
                 f"Received request with version {query.version}, is newer than miner running version {get_version()}. You may updating the repo and restart the miner."
             )
 
+    # TODO: Remove the function after testing
```

Comment:
can be in a seperate script, using specific hotkey wallet to send synapses to validators